template/rust-http/Dockerfile

```diff
@@ -22,6 +22,9 @@ ENV USER=app
 COPY --from=watchdog /fwatchdog /usr/bin/fwatchdog
 RUN chmod +x /usr/bin/fwatchdog
 
+COPY --from=builder /home/rust/main/target/x86_64-unknown-linux-musl/release/main .
+RUN chmod +x ./main
```

Comment:
This doesn't work for me, the container immediately exits. The below code does work though, and follows the same format as fwatchdog, of putting the executable in /usr/bin

`# Copy main binary
COPY --from=builder /home/rust/main/target/x86_64-unknown-linux-musl/release/main /usr/bin/main
RUN chmod +x /usr/bin/main

# Set up watchdog for HTTP mode
ENV fprocess="main"`

Reply:
Confirmed the additional fix from @codelite7 above works.

Reply:
@digitalsanctum want to send a PR that contains this commit and adds the fix by @codelite7 ? I don't want to merge a broken PR, but would merge one that combines both changes, if it results in a working template.

Reply:
Sure thing

Reply:
Thanks for correcting it and sorry for not following up quicker, hadn't been using this. Just pushed the fix though